storage-shared-benchmarking/src/main/java/com/google/cloud/storage/benchmarking/CloudMonitoringResult.java

```diff
@@ -147,6 +147,23 @@ public int hashCode() {
         throughput);
   }
 
+  public String generateCustomMetric() {
+    return String.format(
+        "throughput{library=%s,api=%s,op=%s,object_size=%d,chunksize=%d,workers=%d,crc32c_enabled=%b,md5_enabled=%b,bucket_name=%s,status=%s,app_buffer_size=%d}%.1f",
+        library,
+        api,
+        op,
+        object_size,
+        chunksize,
+        workers,
+        crc32c_enabled,
+        md5_enabled,
+        bucket_name,
+        status,
+        app_buffer_size,
```

Comment:
Now that we're actually rendering the output string for the metrics ingestion, can we change this field names to camel case rather than snake case?